fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePartitioner.java

```diff
@@ -612,10 +678,10 @@ public LuceneRecordCursor getOldestNDocuments(@Nonnull final LucenePartitionInfo
      * @return void future
```

Comment:
TODO: I need to update the javadoc here. It's a future containing the number of documents that were moved.